microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md

```diff
@@ -23,16 +23,17 @@ This represents the WebView2 Environment.
 --------------------------------|---------------------------------------------
 [BrowserVersionString](#browserversionstring) | The browser version info of the current CoreWebView2Environment, including channel name if it is not the stable channel.
 [NewBrowserVersionAvailable](#newbrowserversionavailable) | NewBrowserVersionAvailable fires when a newer version of the Edge browser is installed and available for use via WebView2.
+[CompareBrowserVersions](#comparebrowserversions) | Compare browser versions correctly to determine which version is newer, older or same.
```

Comment:
```suggestion
[CompareBrowserVersions](#comparebrowserversions) | Compare browser versions to determine if they match or are different.
```

Reply:
Can we clarify what 2+ things are being compared? For example, are we comparing the installed version of Microsoft Edge and the Runtime?

Reply:
Any browser version string. One can compare 86.0.622.11 to 87.0.628.0, for instance.

Reply:
Thanks for clarifying. Do we have scenarios when customers would need to use this method? Or perhaps would it be used with some other APIs? That'll help clarify when customers can use this method and in what situation.

Reply:
For example, they could take the result from calling GetAvailableCoreWebView2BrowserVersionString and compare it with the SDK version they want to use. [This](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-622/webview2-idl#comparebrowserversions) is the win32 version for more details.

Reply:
I like that clarification. Can we add that?